core/src/components/range/range-interface.ts

```diff
@@ -8,7 +8,15 @@ export interface RangeChangeEventDetail {
   value: RangeValue;
 }
 
+export interface RangeChangeStartEventDetail {
+  value: RangeValue;
+}
+
+export interface RangeChangeEndEventDetail {
+  value: RangeValue;
+}
```

Comment:
These need to be documented in `readme.md`. You can add them above/under the `RangeChangeEventDetail` interface.